docs/dev/05-plugins.md

```diff
@@ -47,19 +47,20 @@ Karma Framework connects existing testing libraries to Karma's API, so that thei
 results can be displayed in a browser and sent back to the server.
 
 Karma frameworks _must_ implement a `window.__karma__.start` method that Karma will
-call to start test execution. This function is called with an object that has methods
-to send results back to karma:
+call to start test execution. Once this function is called, the `window.__karma__`
+object will contain the following methods which you can use to send results back to 
+karma:
 
-* `.result` a single test has finished
-* `.complete` the client completed execution of all the tests
-* `.error` an error happened in the client
-* `.info` other data (e.g. number of tests or debugging messages)
+* `window.__karma__.result(result: Result)` a single test has finished
```

Comment:
How about placing the definitions of the required object properties under the function calls rather than below. eg
`window.__karma__.result(result)` a single test has finished
  `result` is an object with these properties:
   ...